build_tools/ci/cpu_comparison/matmul_template/matmul_trunci_scaling_MxK_KxN.mlir

```diff
@@ -0,0 +1,27 @@
+// input ${M}x${K}x${TYPE1}
+// input ${K}x${N}x${TYPE1}
+
+func.func @matmul_trunci(%arg0: tensor<${M}x${K}x${TYPE1}>, %arg1: tensor<${K}x${N}x${TYPE1}>) -> tensor<${M}x${N}x${TYPE1}>
+{
+  %cst = arith.constant ${ZERO} : ${TYPE2}
+  %cst_mul = arith.constant 10 : ${TYPE2}
+  %cst_div = arith.constant 137 : ${TYPE2}
+  %0 = tensor.empty() : tensor<${M}x${N}x${TYPE2}>
+  %i8out = tensor.empty() : tensor<${M}x${N}x${TYPE1}>
```

Comment:
i8out, so TYPE1 is i8?

Reply:
Correct. It's a i8->i8 test.

Reply:
If it's always an i8 test then TYPE1 shouldn't be templatized. 

Maybe this test shouldn't be templatized, for example: https://github.com/nod-ai/iree-amd-aie/blob/main/build_tools/ci/cpu_comparison/test_files/matmul_int32.mlir